matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/helper/ChunkEntityHelper.kt

```diff
@@ -236,3 +240,14 @@ internal fun ChunkEntity.isMoreRecentThan(chunkToCheck: ChunkEntity): Boolean {
     // We don't know, so we assume it's false
     return false
 }
+
+internal fun ChunkEntity.Companion.findLatestSessionInfo(realm: Realm, roomId: String): Set<SessionInfoPair>? =
+        ChunkEntity.findLastForwardChunkOfRoom(realm, roomId)?.timelineEvents?.mapNotNull { timelineEvent ->
+            timelineEvent?.root?.asDomain()?.content?.toModel<EncryptedEventContent>()?.let { content ->
+                content.sessionId ?:  return@mapNotNull null
+                content.senderKey ?:  return@mapNotNull null
+                Pair(content.sessionId, content.senderKey)
+            }
+        }?.toSet()
+
+internal typealias SessionInfoPair = Pair<String, String>
```

Comment:
I think it's exposed in the service, so shouldn't be internal. Can it be a simple data class also, to avoid confusion with first/second.

Reply:
if it can be `internal`, the compiler will complain.